rotkehlchen/chain/optimism/types.py

```diff
@@ -1,53 +1,6 @@
 from typing import TYPE_CHECKING, Literal
 
-from rotkehlchen.types import ChainID, ChecksumEvmAddress, EvmTransaction, EVMTxHash, Timestamp
```

Comment:
Remove this entire file. It's empty now, isn't it?

Reply:
it's not empty it has:

```
from typing import TYPE_CHECKING, Literal

if TYPE_CHECKING:
    # This is a list of all optimism abis that are guaranteed to be returned
    # by EthereumContracts as they are stored in the file/DB
    OPTIMISM_KNOWN_ABI = Literal['VELO_V2_LP']
```